activity/permissions.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework import permissions
+
+from workflow.models import ActivityUserOrganizationGroup
+
+
+class IsReadOnly(permissions.BasePermission):
+    def has_permission(self, request, view):
+        if request.method in permissions.SAFE_METHODS:
+            return True
+
+        return not ActivityUserOrganizationGroup.objects.filter(
```

Comment:
It could be written as following so it is more readable
```suggestion
        if user.is_authenticated:
            viewer_group = request.user.activity_user.activity_user_org_group.all().filter(group__name='Viewer')
            if not viewer_group.exists():
                return True
      
        return False
```